xemutest/test_base.py

```diff
@@ -87,6 +107,8 @@ def prepare_config(self):
 			f.write(config)
 
 	def launch_video_capture(self):
+		if not self.test_env.video_capture_enabled:
+			return
```

Comment:
ffmpeg path is not used to launch ffmpeg here

Reply:
Sorry, had this commit split off of my larger pgraph tests PR and forgot to merge down later fixes.